packages/main/src/features/InputSuggestions.js

```diff
@@ -1,5 +1,6 @@
 import { registerFeature } from "@ui5/webcomponents-base/dist/FeaturesRegistry.js";
 import { getI18nBundle } from "@ui5/webcomponents-base/dist/i18nBundle.js";
+import { escapeHTML, generateHighlightedMarkup } from "src/utils/Strings.js";
```

Comment:
should be`import { escapeHTML, generateHighlightedMarkup } from "../utils/Strings.js";`